forge-gui/res/cardsfolder/upcoming/vladimir_and_godfrey.txt

```diff
@@ -0,0 +1,10 @@
+Name:Vladimir and Godfrey
+ManaCost:2 W
+Types:Legendary Creature Zombie Knight
+PT:3/2
+A:AB$ ChangeZone | Cost$ 2 W | Origin$ Graveyard | Destination$ Battlefield | Defined$ Self | ActivationZone$ Graveyard | IsPresent$ Creature.YouCtrl+powerEQ1+toughnessEQ1 | Tapped$ True | SubAbility$ DBEffect | SpellDescription$ Rejuvenation - Return CARDNAME from your graveyard to the battlefield tapped. It perpetually gets +1/+1. Activate only if you control a 1/1 creature.
```

Comment:
Take `Rejuvenation –` from SpellDescription and make it `PrecostDesc$ Rejuvenation –`

See stuff like Blink Dog